### ISSUE TITLE
Bug in Roller Setup: RollApp DA Light Client Fails to Start Properly and set up relayer

### ISSUE THREAD

**Issue by ferianto-dropid** [dymensionxyz/roller#1039]
### Roller version

💈 Roller version 1.6.4-alpha-rc06 💈 Build time: 2024-09-22T13:16:54Z 💈 Git commit: 8cd0106867840db19287846ed1387a1b8bd9879e

### Rollpp type (EVM/WASM) and version

v2.2.0-rc01-51-gfe4246e

### OS

PRETTY_NAME="Ubuntu 22.04.5 LTS" NAME="Ubuntu" VERSION_ID="22.04" VERSION="22.04.5 LTS (Jammy Jellyfish)" VERSION_CODENAME=jammy ID=ubuntu ID_LIKE=debian HOME_URL="https://www.ubuntu.com/" SUPPORT_URL="https://help.ubuntu.com/" BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/" PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy" UBUNTU_CODENAME=jammy

### Processor Architecture

x86_64

### Others

  CPU op-mode(s):        32-bit, 64-bit
  Address sizes:         40 bits physical, 48 bits virtual
  Byte Order:            Little Endian
CPU(s):                  2
  On-line CPU(s) list:   0,1
Vendor ID:               GenuineIntel
  Model name:            DO-Regular
    CPU family:          6
    Model:               79
    Thread(s) per core:  1
    Core(s) per socket:  2
    Socket(s):           1
    Stepping:            1
    BogoMIPS:            4988.27
 total        used        free      shared  buff/cache   available
Mem:           3.8Gi       389Mi       415Mi       4.0Mi       3.0Gi       3.2Gi
Swap:             0B          0B          0B

### Steps to reproduce it

Run the RollApp setup using roller rollapp setup.
Attempt to start the RollApp sequencer and DA light client using roller rollapp services start.
Attempt to check health and status of the RollApp.
Check DA light client logs in /root/.roller/da-light-node/light_client.log.

### Expected result

The RollApp sequencer and DA light client should start smoothly, and the roller rollapp services logs should show activity in both logs, indicating proper DA light client function.

### Actual result

The RollApp sequencer starts correctly, but the DA light client does not generate logs, causing a failure in syncing. Logs show waiting for the light client file to appear, but it never does. The balance of the DA wallet shows 0 utia, which may be related.

### Log Files

-bash: /root/.roller/rollapp/rollapp.log: Permission denied
-bash: /root/.roller/da-light-node/light_client.log: No such file or directory

### Notes

Setup Relayer:

Steps Taken During Relayer Setup:
Ran roller relayer setup command to establish IBC connection.
Encountered error: ERROR rollapp did not reach valid height: timeout waiting for rollapp height to reach 1.
Adjusted dymint block time settings as per log message.
Observed Issues During Relayer Setup:
Logs indicated a long wait for RollApp height to reach 1, which delayed the relayer setup.
RollApp services were running (roller rollapp services start showed success), but the RollApp height failed to progress.
The issue might be related to the DA wallet balance, as the DA address has 0 utia.
Tried checking Celestia connection and restarting services, but the issue persists

**Comment by ferianto-dropid** [comment]
During the relayer setup process, I encountered an issue where the RollApp height did not progress as expected. This caused the roller relayer setup to timeout while waiting for the RollApp height to reach 1. Although I verified that the RollApp sequencer was running, the height remained static.

I adjusted the dymint block time settings as indicated in the logs, but the issue persisted. It's possible that the system may be running low on resources such as CPU or RAM, which could be slowing down the process. I've attached relevant system resource checks and logs to provide more context.

Any guidance or recommendations for troubleshooting this would be greatly appreciated!
